lib/GenomeAnnotationAPI/utils.py

```diff
@@ -0,0 +1,56 @@
+import logging
+
+
+class Utils:
+    def __init__(self, workspace_client, services):
```

Comment:
It looks like most of the urls aren't used and I'm worried that in the future someone will try to initialize using the wrong token.  So maybe drop the unused ones for now.